test/it/code-samples/positive/pages/pageElements.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
What this is for?

Reply:
From what I gathered tests on page context require a pageElement file.
I don't need any elements so that's an empty Json

Reply:
Cool, I got lost with the naming convention here, didn't understand you had added this file for all pages tests...